### PR TITLE
[120] Add technologies to a freak

### PR DESCRIPTION
**Pull request by salcudeann**
## Description
Create a Technology model and a database table named `technologies` for this model.
Create an association between Freak and Technology and a database table for this association named `freak_technologies` which helps us to make the association between these 2 models.
Also, the json data from technologies was removed,  now when we run the `freaks_spec` test a technology object is created in database and assigned for a freak.

## Related

[[120] Add Technologies to a Freak ](https://trello.com/c/uCnM0Nrv/120-add-technologies-to-a-freak)

## Steps to Test or Reproduce

The seeds.rb file will populate tables `technologies`, `freaks` and `freak_technologies` with some data. You can run next GraphQl query to get all freaks and see that data from technologies will be taken from database
```GraphQl

query Freaks($last: Int, $before: String) {
    freaks(last: $last, before: $before) {
        edges {
            node {
                projects {
                    technologies {
                        id
                        name
                        description
                    }
                }
                technologies {
                    id
                    name
                    description
                }
            }
        }
    }
}

```

### CODE REVIEW

**Comment by salcudeann** [comment]
[Add Technologies to a Freak](https://trello.com/c/uCnM0Nrv/120-add-technologies-to-a-freak)

**Review by gabrielgiroe1 — COMMENTED** [review]
Do you need technologies.json file?

File: db/seeds.rb [hunk 1]
```diff
@@ -7,3 +7,14 @@
...
+Technology.create(name: 'Java', description: 'Java is a popular language')
+technology = Technology.create(name: 'Ruby', description: 'Ruby is a popular language')
+
+Freak.create(
+  first_name: 'John',
+  last_name: 'Doe',
+  email: 'freak@gmail.com',
+  description: 'A freak',
+  technologies: [technology]
+)
```
Comment (gabrielgiroe1) [inline db/seeds.rb:20]: You need a test for seed data

**Review by gabrielgiroe1 — CHANGES_REQUESTED** [review]

**Review by gabrielgiroe1 — APPROVED** [review]

File: .rubocop.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ AllCops:
     - vendor/**/*
     - app/controllers/graphql_controller.rb
     - config/database.yml
+    - spec/tasks/seeds_spec.rb
+
+
```
Comment (alex-damian-negru) [inline .rubocop.yml:16]: Some blank lines here.

File: spec/factories/technology.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :technology do
+    name { 'Ruby' }
```
Comment (alex-damian-negru) [inline spec/factories/technology.rb:5]: Not a requirement, but if you wanna be sassy you can use `Faker::ProgrammingLanguage.name`

```ruby
Faker::ProgrammingLanguage.name #=> "Ruby"
```

**Review by alex-damian-negru — APPROVED** [review]
Looks good. Check the comments below before merging.

**Pull request merged**